src/applications/travel-pay/util/string-helpers.js

```diff
@@ -0,0 +1,9 @@
+import { camelCase, lowerCase, startCase, upperFirst } from 'lodash';
+
+export const toPascalCase = str => {
+  return startCase(camelCase(str)).replace(/ /g, '');
```

Comment:
This looks like it's
1. camelCasing the string (removes spaces, camelizes based on space positions)
2. startCasing the camelized string (adds spaces based on capital letter positions)
3. removing the spaces

Could we achieve the same result with this?
`upperFirst(camelCase(str))`